sqlstore/src/main/resources/db/migration/V88977_watch.sql

```diff
@@ -0,0 +1,3 @@
+DROP TABLE Pool_Watcher;
+DROP TABLE Project_Watcher;
+DROP TABLE Run_Watcher;
```

Comment:
Should also drop from the `_Group` table where `name` in (RunWatchers, ProjectWatchers, PoolWatchers)